packages/e2e-tests/src/specs/editor/floatingMenu.js

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import {
+    createNewStory,
+    waitForCarousel
```

Comment:
I'm wondering what went wrong with the pre-commit hook/prettier here 🤔  (also there is `no-unused-vars` and `import/named` error).

Can you check if `npm run lint:js:fix` fixes the issues on your side?